docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,267 @@ include::includes/global-attributes.asciidoc[]
-
-We'll use the clone function to add a new switch (the AeroHive AP) to the PacketFence database. With the mouse pointer, go over the default switch and you'll see a set of icons appearing at the left of it. Click on the second one (a paper sheet with a plus sign).
-
-Configure the switch like the following:
-
-image::docs/images/aerohive-pf-config-webauth.png[scaledwidth="100%",alt="General switch settings"]
-
-The section Roles should be empty while doing webauth.
-
-image::docs/images/aerohive-pf-config-snmp.png[scaledwidth="100%",alt="SNMP switch settings"]
-
-Under the tab RADIUS, please set your RADIUS secret.
-
-Click 'Add switch'
-
-The newly added access point should show up in the list.
-
-PacketFence configuration is done. You may now reboot the demonstration PC (leave the USB key in). Once rebooted, the web browser should open in the PacketFence web administration interface.
 
-Step 3: Configure Aerohive
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-This section will help you configure your Aerohive AP for integration with PacketFence using the VLAN-based enforcement.
+We will nowneed to create a new switch, in PacketFence to be able to manage the terminals behind the Aerohive APs.
 
-Connect APs and Controller
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-First thing to do is to connect the HiveManager. Like you saw in Step 2, make sure to connect the AP on the TRUNK port configured for it.
+Logon to your PacketFence interface::
+* From _Configuration / Policies and Access Control / Switches /_
+* On the line where there is the _default_, on the right, Click on _CLONE_.
 
-Provision the HiveManager
-^^^^^^^^^^^^^^^^^^^^^^^^^
-By default, the HiveManager might be empty, add your AP via its serial number. If necessary update the firmware of the HiveManager and the AP.
 
-NOTE: Please refer to the 'Aerohive Quick Devices Start Guide' to add devices: http://docs.aerohive.com/330000/docs/help/english/documentation/AerohiveDevicesQuickStart.pdf
+image::docs/images/pfaerohive09.png[scaledwidth="100%",alt="New switch configuration",align=right]
 
-NOTE: Please refer to the following guide to learn more about the HiveManager: http://docs.aerohive.com/330000/docs/help/english/documentation/Aerohive_HiveManagerNG_GettingStartedGuide.pdf
+'''
 
-VLAN Enforcement
-^^^^^^^^^^^^^^^^
-Setup the SSIDs
-^^^^^^^^^^^^^^^
-In this guide, we will configure one open SSID using RADIUS MAC-based filtering and one secured SSID using WPA2 Enterprise.
 
-Login to your HiveManager using the admin credentials. You should see the Dashboard. Click on the 'Configure' pane, and follow the following steps.
+In _Definition_ ::
+* _IP Address/MAC Address/Range (CIDR)_, give the network adress of your _Production_ network,for us, it will be *172.16.1.0/24*
+* _Description_, give a description so you can quickly see what it is about,
+* From the _Type_ list, select *AeroHIVE AP*.
```

Comment:
If in our list we mention AeroHIVE, prepare a *separate* PR to fix this.

Reply:
Ok